tool/tsh/tsh_test.go

```diff
@@ -2245,6 +2245,73 @@ func TestEnvFlags(t *testing.T) {
 			},
 		}))
 	})
+
+	t.Run("tsh ssh session env", func(t *testing.T) {
```

Comment:
could add a case to test for an error on headless login without explicit username

Reply:
This is covered in `TestSSHHEadless`